### PR TITLE
feat: remove need to add load_in* during merge

### PR DESCRIPTION
**Pull request by NanoCode012**
Removes need to set these option to False via cli.
- `load_in_8bit`
- `load_in_4bit`
- `flash_attention` due to loading without GPU

### CODE REVIEW

**Review by hamelsmu — APPROVED** [review]

File: src/axolotl/cli/merge_lora.py [hunk 1]
```diff
@@ -18,7 +18,15 @@ def do_cli(config: Path = Path("examples/"), **kwargs):
...
     )
     parsed_cli_args.merge_lora = True
-    parsed_cfg = load_cfg(config, merge_lora=True, **kwargs)
+
+    parsed_cfg = load_cfg(
+        config,
+        merge_lora=True,
+        load_in_8bit=False,
+        load_in_4bit=False,
+        flash_attention=False,
```
Comment (hamelsmu) [inline src/axolotl/cli/merge_lora.py:27]: What's the thing about `flash_attention` and LorA merges?

Comment (hamelsmu) [reply src/axolotl/cli/merge_lora.py:27]: Nevermind I just read [this comment](https://github.com/OpenAccess-AI-Collective/axolotl/issues/1002#issuecomment-1870358425) which explains everything

**Comment by hamelsmu** [comment]
This PR closes #1002 , I believe

**Comment by NanoCode012** [comment]
> This PR closes #1002 , I believe

I think that person's issue was related to max_memory(?) The corresponding fix should be in PR #918

**Comment by hamelsmu** [comment]
> I think that person's issue was related to max_memory(?) The corresponding fix should be in PR https://github.com/OpenAccess-AI-Collective/axolotl/pull/918

Oh I see sorry about that, I'll unlink

**Pull request merged**